### PR TITLE
Users/profile

### PR DESCRIPTION
**Pull request by tubhutani13**

### CODE REVIEW

File: lib/tasks/admin.rake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    password = STDIN.gets.chomp
+    puts "Enter Password Confirm"
+    password_confirmation = STDIN.gets.chomp
+
+    user = User.create(
+      :name => name.to_s,
+      :email => email.to_s,
+      :password => password.to_s,
+      :password_confirmation => password_confirmation.to_s,
+      :verified => true,
```
Comment (shubhamjuneja1) [inline lib/tasks/admin.rake]: aren't we saving verified_at instead of verified?

File: app/models/user.rb [hunk 2]
```diff
@@ -5,8 +5,15 @@ class User < ApplicationRecord
        }
 
   has_secure_password
+  has_one_attached :profile_picture do |attachable|
```
Comment (shubhamjuneja1) [inline app/models/user.rb]: dependent: destroy

File: app/models/user.rb [hunk 3]
```diff
@@ -5,8 +5,15 @@ class User < ApplicationRecord
...
+    attachable.variant :thumb, resize_to_limit: [100, 100]
```
Comment (shubhamjuneja1) [inline app/models/user.rb:9]: extract method

File: app/helpers/users_helper.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+module UsersHelper
+  def user_profile_picture_link(user)
+    user.profile_picture.present? ? rails_blob_path(user.profile_picture.variant(:thumb), disposition: "attachment") : "default.jpeg"
+  end
+
+  def change_password
+    current_user.generate_change_password_token
```
Comment (shubhamjuneja1) [inline app/helpers/users_helper.rb]: why this is in helper????????????????

**Review by shubhamjuneja1 — CHANGES_REQUESTED** [review]
Changes required

File: app/models/user.rb (hunk 3 above)
Comment (tubhutani13) [reply app/models/user.rb:9]: discuss

**Pull request closed without merging**